components/credentials/build.gradle

```diff
@@ -9,6 +9,7 @@ buildscript {
     }
     dependencies {
         classpath("org.springframework.boot:spring-boot-gradle-plugin:$springBootVersion")
+        classpath("com.google.protobuf:protobuf-gradle-plugin:0.9.4")
```

Comment:
I am sure that there was a reason why this dependency with this version was added. The commit message does not explain, so I do not know what it was. I am fine with the change as it seems solved the problem.

Reply:
I don't have a good understanding of how protobuf works, I just know that these were the minimal changes I could come up with to resolve the build errors.